collect_app/src/test/java/org/odk/collect/android/external/FormUriActivityTest.kt

```diff
@@ -709,6 +711,16 @@ class FormUriActivityTest {
         assertStartSavedFormIntent(null, instance.dbId, true)
     }
 
+    @Test
+    fun `The activity should be finished after clicking the back button when an error dialog is displayed`() {
```

Comment:
Why do we need this test when all the other tests now assert on the cancellation behaviour?